_help/troubleshooting.md

```diff
@@ -325,3 +325,29 @@ Check your `ulimit -a`. Many systems have a 1024 open file descriptor limit by d
 ```
 
 Make sure to raise your ulimit. Example: `ulimit -n 16384`
+
+## Headless TCP Services Losing Connection from Istiofied Containers
+
+Envoy is reset every 15 minutes by the `istio-ca` deployment, as a part
+of this process it will restart Envoy sidecars causing them to
+disconnect TCP streams. In order to stop this from happening, you will
```

Comment:
I would insert something like:
"While your TCP services should be resilient to rare connection close events (as your pods may move for a number of reasons instance), if you really need to keep long running connection, and until we release the version of istio that doesn't require envoy restart for certificate reload, you will have to disable mTLS [...]"
(though that's kind of a loooong sentence)

Reply:
@ldemailly 30 seconds isn't a long TCP connection. DB connections often stay open for hours by default before being recycled. Will add something about connection resilience as well.